extensions/stackdriver/stackdriver.cc

```diff
@@ -206,8 +206,15 @@ void StackdriverRootContext::onTick() {
   }
   if (enableEdgeReporting()) {
     auto cur = static_cast<long int>(getCurrentTimeNanoseconds());
-    if ((cur - last_edge_report_call_nanos_) > edge_report_duration_nanos_) {
-      edge_reporter_->reportEdges();
+    if ((cur - last_edge_report_call_nanos_) >
```

Comment:
Is it right to using one `last report time` for two types of duration? Isn't that the longer duration one will never be triggered?

Reply:
updated logic.